src/qc.rs

```diff
@@ -5,38 +5,40 @@
 
 use std::boxed::Box;
 use std::fmt;
+use core::{f32, f64};
 
 use quickcheck::{Arbitrary, Gen};
 
 use int::LargeInt;
+use float::Float;
 
 // Generates values in the full range of the integer type
 macro_rules! arbitrary {
-    ($TY:ident : $ty:ident) => {
+    ($id:ident: $ty:ty) => {
```

Comment:
I would prefer to maintain the status quo for now just to reduce the noise in the diff. I don't have a strong preference for the name of the macro arguments.